ui/completion.py

```diff
@@ -180,7 +210,7 @@ def __init__(self, view: sublime.View) -> None:
 
     @property
     def popup_content(self) -> str:
-        syntax = self.view.syntax() or sublime.find_syntax_by_name("Plain Text")[0]
+        syntax = self.view.syntax() or PLAIN_TEXT_SYNTAX
```

Comment:
Maybe `view.syntax() or PLAIN_TEXT_SYNTAX` can be extracted to a helper method, `get_view_syntax`.  
We just use it quite often

Reply:
Done in [50cb36d](https://github.com/TheSecEng/LSP-copilot/pull/18/commits/50cb36dd15f1300424a21df184af5bda2e6e1b9f)